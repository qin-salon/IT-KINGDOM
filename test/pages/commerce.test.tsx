import CommercePage from "src/pages/support/commerce";
import { render } from "test/test-utils";

describe("CommercePage", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<CommercePage />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
