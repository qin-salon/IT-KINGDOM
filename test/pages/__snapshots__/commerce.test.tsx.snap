// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommercePage matches snapshot 1`] = `
<DocumentFragment>
  <header
    class="container"
  >
    <h1>
      <a
        href="/"
      >
        <div
          style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
        >
          <div
            style="box-sizing: border-box; display: block; max-width: 100%;"
          >
            <img
              alt=""
              aria-hidden="true"
              role="presentation"
              src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTgwIiBoZWlnaHQ9IjM4IiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZlcnNpb249IjEuMSIvPg=="
              style="max-width: 100%; display: block;"
            />
          </div>
          <img
            alt="IT KINGDOM logo"
            decoding="async"
            src="/_next/image?url=%2Fimg%2Flogo.png&w=768&q=75"
            srcset="/_next/image?url=%2Fimg%2Flogo.png&w=320&q=75 1x, /_next/image?url=%2Fimg%2Flogo.png&w=420&q=75 2x, /_next/image?url=%2Fimg%2Flogo.png&w=768&q=75 3x"
            style="visibility: visible; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
          />
        </div>
      </a>
    </h1>
    <nav
      class="sansSerif nav"
    >
      <ul>
        <li>
          <a
            href="/faq"
          >
            よくある質問
          </a>
        </li>
        <li>
          <a
            href="https://qin.salon/signup"
            rel="noopener noreferrer"
            target="_blank"
          >
            入国手続
          </a>
        </li>
      </ul>
    </nav>
  </header>
  <main
    class="container styled"
  >
    <h1>
      特定商取引法に基づく表記
    </h1>
    <p>
      「特定商取引に関する法律」第11条に基づき、以下のとおり表示致します。
    </p>
    <h2>
      事業者の名称
    </h2>
    <p>
      しまぶー
    </p>
    <h2>
      事業者の所在地
    </h2>
    <p>
      東京都豊島区南大塚2-11-10 ミモザビル3F
    </p>
    <h2>
      事業者の連絡先
    </h2>
    <p>
      dev@qin.salon
    </p>
    <p>
      営業時間: 平日10:00〜19:00
    </p>
    <h2>
      販売価格について
    </h2>
    <p>
      販売価格は、表示された金額（表示価格/消費税別）と致します。
    </p>
    <h2>
      販売価格以外でお客様に発生する金銭
    </h2>
    <p>
      当サイトのページ閲覧等に必要となるインターネット接続料金、通信料金等はお客様の負担となります。それぞれの料金は、お客様がご利用のインターネットプロバイダまたは携帯電話会社にお問い合わせください。
    </p>
    <h2>
      支払方法
    </h2>
    <p>
      クレジットカードの他、Stripeが指定するお支払い方法をご利用いただけます。
    </p>
    <h2>
      代金の支払時期:
    </h2>
    <p>
      お客様が選択した支払い方法に関する、プラットフォーム事業者、クレジットカード会社等の規約に基づきます。
    </p>
    <h2>
      役務または商品の引渡時期
    </h2>
    <p>
      購入に関するページに特別な定めを置いている場合を除き、代金決済手続きの完了確認後、直ちにご利用いただけます。
    </p>
    <h2>
      返品・キャンセルに関する特約
    </h2>
    <p>
      本サイトで販売するサービスについては、購入手続完了後の返品またはキャンセルをお受けいたしません。なお、サービスに瑕疵がある場合は、利用規約の定めに従って対応します。特別な販売条件または提供条件がある商品またはサービスについては、各サービスの購入ページにおいて条件を表示します。
    </p>
    <time>
      2020年10月15日 制定
    </time>
  </main>
  <footer
    class="sansSerif container"
  >
    <ul
      class="links"
    >
      <li
        class="link"
      >
        <a
          href="/support/terms"
        >
          利用規約
        </a>
      </li>
      <li
        class="link"
      >
        <a
          href="/support/privacy-policy"
        >
          プライバシーポリシー
        </a>
      </li>
      <li
        class="link"
      >
        <a
          href="/support/commerce"
        >
          特定商取引法に基づく表記
        </a>
      </li>
      <li
        class="link"
      >
        <a
          href="https://docs.google.com/forms/d/e/1FAIpQLSf7lYhGHmnl2KTPXsArv5KCoZvlwX7pMF7EZrDXSgWQWnJDhw/viewform"
          rel="noopener noreferrer"
          target="_blank"
        >
          お問い合わせ
        </a>
      </li>
    </ul>
    <small
      class="copyright"
      lang="en"
    >
      @ 2020 IT KINGDOM
    </small>
  </footer>
</DocumentFragment>
`;
